server/app/boards.go

```diff
@@ -250,20 +250,67 @@ func (a *App) CreateBoard(board *model.Board, userID string, addMember bool) (*m
 		if addMember {
 			a.wsAdapter.BroadcastMemberChange(newBoard.TeamID, newBoard.ID, member)
 		}
+		if newBoard.ChannelID != "" {
```

Comment:
Won't this send again the `member` membership? I think we could fix this moving the `if addMember` condition to be the `else` of this line

Reply:
Good point, the answer is yes, not a big deal, but can be improved, let me fix it.

Reply:
Done